edgelet/docker-rs/Cargo.toml

```
@@ -11,10 +11,10 @@ hyper = { version = "0.14", features = ["client", "http1"] }
 serde = "1"
 serde_derive = "1"
 serde_json = "1"
-tokio = { version = "1", features = ["time"] }
+tokio = { version = "1", features = ["parking_lot", "time"] }
```

```suggestion
tokio = { version = "1", features = ["time"] }
```
This crate only uses `tokio::time::..`